projectmanagement-domain/src/main/java/life/qbic/projectmanagement/application/authorization/authorities/QBiCUserAuthorityService.java

```diff
@@ -12,19 +11,19 @@
  * Provides granted authorities given a user
  */
 @Service
-public class QBiCUserAuthorityService implements UserAuthorityProvider {
+public class QbicUserAuthorityService implements UserAuthorityProvider {
 
   private final UserRoleRepository userRoleRepository;
 
-  public QBiCUserAuthorityService(
+  public QbicUserAuthorityService(
       @Autowired UserRoleRepository userRoleRepository
   ) {
     this.userRoleRepository = userRoleRepository;
   }
 
   @Override
-  public List<GrantedAuthority> getAuthoritiesByUserId(UserId userId) {
-    List<UserRole> roles = userRoleRepository.findAllByUserId(userId.get());
+  public List<GrantedAuthority> getAuthoritiesByUserId(String userId) {
```

Comment:
comprehension question: what is the the thought behind replacing this Id class by a String and where should it be done in general? E.g. we are still using ProjectId in some other classes.

Reply:
Is it for ease of use of services?